prime-dotnet-webapi/ViewModels/Paper Enrollees/EnrolleeLinkedEnrolmentViewModel.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+namespace Prime.ViewModels.PaperEnrollees
+{
+    public class EnrolleeLinkedEnrolmentViewModel
+    {
+        public int? EnrolleeId { get; set; }
```

Comment:
Don't think EnrolleeId has to be nullable in the VM if its not nullable in the model.

Reply:
Fixed